tools/setup_presubmit_repos.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+#
+# Copyright 2025 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import pathlib
+import sys
+import shutil
+import os
+import logging
+
+import bcr_presubmit
+
+
+def get_host_platform():
+    if sys.platform == "darwin":
+        return "macos"
+    if sys.platform == "win32":
+        return "windows"
+    # Default to linux
+    return "linux"
+
+
+def get_task_platform(task):
+    platform = task.get("platform")
+    if platform.startswith("macos"):
+        return "macos"
+    if platform.startswith("windows"):
+        return "windows"
+    # Default to linux
+    return "linux"
+
+
+def print_build_instruction(module_name, module_version, repo_root, task_configs):
+    build_targets = test_targets = bazel_version = None
+
+    # Find the first task that matches the host platform
+    host_platform = get_host_platform()
+    task_found = False
+    for task in task_configs["tasks"].values():
```

Comment:
should we maybe print the name of the task we "chose"?

Reply:
Good idea, done!